<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettings</class>
 <widget class="QWidget" name="FormSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_labelLivestreamerCommands">
     <property name="text">
      <string>Livestreamer commands</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_textboxFullLivestreamerCommand">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_labelAddBookmark">
     <property name="text">
      <string>Add bookmark by link</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_textboxAddBookmark">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="placeholderText">
      <string>http://www.panda.tv/1234</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_buttonAddBookmark">
     <property name="text">
      <string>Add bookmark</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_checkboxMinimizeToTray">
     <property name="text">
      <string>Minimize to tray</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>226</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
